rfcs/text/0010/email.yml

```diff
@@ -0,0 +1,118 @@
+---
+- name: email
+  title: Email
+  group: 2
+  short: Fields describing an email message.
+  description: >
+    Email fields are used for information about an email message header and body.
+  type: group
+  fields:
+
+    - name: to
+      level: extended
+      type: keyword
+      short: Recipient address(es)
+      description: >
+        Stores the `to` email address(es).
+      example: "employee@example.com"
+      normalize:
+        - array
+
+    - name: from
+      level: extended
+      type: keyword
+      short: Sender address(es)
+      description: >
+        Stores the `from` email address(es).
+      example: "administrator@example.com"
+      normalize:
+        - array
+
+    - name: cc
```

Comment:
Shouldn't cc and bcc be arrays, assuming plurality of values?

Reply:
I have the field definition defined as an array (`normalize: array`), but I didn't use an array for the example value. Will correct.